ncm-opennebula/src/main/resources/tests/profiles/vm.pan

```diff
@@ -230,3 +230,5 @@ prefix "/system/opennebula";
     "cores", 1,
 );
 
+"machine" = "pc-q35-rhel9.4.0";
```

Comment:
Shouldn't the output of a tests also be changed?

Reply:
@wpoely86  indeed, I have included the missing regex

Reply:
the regex turned out to be not very strict. Fixed also.

> 
> ah ok why it was working before (actually I also forgot to include the check) is because the test itself the regex is not quite strict as you can see it was:
> ```
> ^MEMORY\s?=\s*"\d+"\s*$
> ^OS\s?=\s?\[$
> ^\s*BOOT\s?=\s*""\s*$
> ^RAW\s?=\s?\[$
> ```
> 
> so this also work if I include something after OS = [    so the regex was working indeed before, now I have included the regex explicitly so if you modify MACHINE it will fail
> so the previous check only was looking for BOOT after OS and indeed it was right.. even if I include MACHINE between... now its more explicit